public/data/sunil_joshi.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Sunil",
+  "bio": "Open Source Enthusiast",
+  "avatar": "https://github.com/iamsunil25.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/iamsunil25",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/iamsunil25",
+      "icon": "twitter"
+    }
+  ]
+ 
```

Comment:
Please can you remove this line `17` @iamsunil25 ? Thank you

Reply:
yes i am removing

Reply:
@CodingSpecies i removed line 17

Reply:
Thank you @iamsunil25

Reply:
Please mark this as resolved if resolved